app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if "cp" not in command:  # перевірка на наявність команди cp в command
+        return
```

Comment:
It will not work properly:
for example: command == "hm copy.txt cpd.txt"
Will return true
And it will be wrong because command is hm